packages/web/src/components/share-modal/utils.ts

```diff
@@ -13,8 +14,17 @@ import { messages } from './messages'
 
 type ShareToTwitterEvent = Omit<ShareToTwitter, 'eventName' | 'source'>
 
-const getShareHandle = async (handle: string) => {
-  const twitterHandle = await getTwitterHandleByUserHandle(handle)
+const getTwitterShareHandle = async ({
+  handle,
+  twitterHandle
+}: {
+  handle: string
+  twitterHandle: Nullable<string>
+}) => {
+  if (!twitterHandle) {
```

Comment:
Same question here

Reply:
same answer as above :)
was removed, and re-added after our convo. should be removed again. maybe should be removed now? didnt think too much about it. i can check

Reply:
handled